nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryAPI.java

```diff
@@ -4017,6 +4062,38 @@ public void testPlacelessDocument() {
         session.save();
     }
 
+    /*
+     * NXP-28727
+     */
+    @Test
+    public void testCreateDocumentUnderPlacelessDoc() {
+        DocumentModel parentFile = session.newDocumentModel(null, "parentFile", "File");
```

Comment:
```suggestion
        DocumentModel parent = session.newDocumentModel(null, "parent", "File");
```
`parent` and `child`, simpler,  like in the other test method you added.